contrib/win32/openssh/OpenSSHBuildHelper.psm1

```diff
@@ -345,20 +351,24 @@ function Package-OpenSSH
             New-Item -ItemType Directory $DestinationPath -Force | Out-Null
         }
         Copy-Item -Path $packageDir\* -Destination $DestinationPath -Force -Recurse
+        Write-Host 'Copied payload to' $DestinationPath
```

Comment:
Write-BuildMsg -AsInfo -Message "Copied payload to $DestinationPath". The message will write to host and log.